pkg/workflows/wasm/host/module.go

```diff
@@ -92,7 +128,7 @@ type ModuleConfig struct {
 	InitialFuel               uint64
 	Logger                    logger.Logger
 	IsUncompressed            bool
-	Fetch                     func(ctx context.Context, req *wasmpb.FetchRequest) (*wasmpb.FetchResponse, error)
+	Fetch                     func(ctx context.Context, req *wasmdagpb.FetchRequest) (*wasmdagpb.FetchResponse, error)
```

Comment:
can just be removed from struct, no?

Reply:
Not if we want backwards compatibility. We need to support the DAG code until nothing uses it.

Reply:
had we considered creating a `host/v2/module` and redefine the `Module` to be v2 only?  All legacy code would use the legacy struct